### DISCUSSION TITLE
missing run once event

### DISCUSSION THREAD

**Discussion by serveryang** [jhuckaby/Cronicle#777] [Q&A]
mssing set `Event Timing Object` to null in web ui,  i want to run event only once.

**Comment by jhuckaby** [comment]
I don't really understand what you're saying.  If you want to run an event only once, set the year, month, day, hour and minute selectors.

Moving this to a discussion.

**Comment by serveryang** [comment]
if year, month, day, hour and minute set , then the event will run repeat by `Event Timing Object`'s setting.
my mean is , if i want run a command only once(when command excute finised, notify by web hook), not repeat to run event. how to set `Event Timing Object`?

**Reply by jhuckaby** [reply]
I have absolutely no idea, sorry.  I'll leave this discussion thread open, in case anyone else in the community can help.  Apologies.

**Reply by serveryang** [reply]
ok
